src/applications/vre/28-8832/config/chapters/dependent-information/dependentAddress.js

```diff
@@ -0,0 +1,44 @@
+import emailUI from 'platform/forms-system/src/js/definitions/email';
+import {
+  buildAddressSchema,
+  addressUISchema,
+} from '../../../../../disability-benefits/686c-674/config/address-schema';
+
+const dependentAddress = buildAddressSchema(true);
+// reset boolean type for checkbox
+dependentAddress.properties['view:livesOnMilitaryBase'] = {
+  type: 'boolean',
+};
+
+export const schema = {
+  type: 'object',
+  properties: {
+    dependentAddress,
+    phoneNumber: {
+      type: 'string',
+      minLength: 7,
+      maxLength: 10,
+    },
+    emailAddress: {
+      type: 'string',
+      minLength: 6,
+      maxLength: 80,
+      pattern:
+        '^[_A-Za-z0-9-]+(\\.[_A-Za-z0-9-]+)*@[A-Za-z0-9-]+(\\.[A-Za-z0-9]+)*(\\.[A-Za-z]{2,})$',
+    },
+  },
+};
+
+export const uiSchema = {
+  dependentAddress: addressUISchema(true, 'dependentAddress', () => true),
```

Comment:
@cohnjesse - did you see my above comment? I think you should be able to pass in a callback such as `isDependent` since it's just getting assigned to `ui:required` which has formData in its scope.